EIPS/eip-epbs.md

```diff
@@ -0,0 +1,260 @@
+
+### Three state transition functions
+
+The current EIP adds an extra state transition function to the block processing in Ethereum. Processing a `SignedBeaconBlock` changes the consensus layer `BeaconState`. A `SignedExecutionPayloadEnvelope` changes both the execution layer state and the consensus layer one. As such, the envelope commits to the consensus layer post-state-transition beacon state root. 
+
+### Compatible designs 
+
+#### Inclusion lists
+
+This EIP is fully compatible with forward inclusion lists as specified in [EIP-7547](./eip-7547.md) or similar.
+
+#### Slot auctions
+
+A simple change to this EIP is to remove the blockhash commitment from the `SignedExecutionPayloadHeader`. This allows the builder to commit any payload to the slot. A preliminary security analysis shows that payload equivocation does not weaken forkchoice's FFG. Some advantages of Slot auctions include:
+
+- Better user experience as any submitted transaction can be included in the next block (with block auctions a transaction sent in the first half of the slot can only be included in the following block). 
+- Longer continuous time to build blocks. 
+- Better compatibility designs with forkchoice enforced inclusion list proposals. 
+
+
+## Backwards Compatibility
+
+This EIP introduces backward incompatible changes to the block validation rule set on the consensus layer and must be accompanied by a hard fork. 
+
+## Security Considerations
+
+### Reorg resistance 
+
+`PROPOSER_SCORE_BOOST` is reduced from 40 to 20. This does not allow however a en-anti reorg by a proposer with 20% of the total stake as the attacker's payload will also be not included. Proposers are protected against *1-slot sandwich* reorgs by a colluding set of validators and builders controlling more than 20% of the total stake
+
+### Builder safety
+
+- A colluding set of proposers and attesters controlling consecutive blocks and more than 20% of the total stake can reorg a builder's payload and force it to pay the bid's value. 
+- There is no possible *unbundling* of the builder's payload, that is, if the builder reveals a payload for the head of the chain, no other payload is possible for the current slot. 
+
+### Malicious PTC
+
+The expected time for a malicious attacker, controlling 35% of the total stake, to have a majority control on the PTC is 205 000 years. 
```

Comment:
Why choose 35%?

Reply:
Because to include a curve it would require me to add a pdf file to the assets. Didn't seem particularly important, should I?

Reply:
Ah no I think it's fine but just wondering if 35% holds some specific importance. Elsewhere you mention 20% adversary for the proposer boost, 33.33333...% is known to be a critical threshold too.

Reply:
Ahh right, the 20% for proposer boost is actually computed, the argument is in https://ethresear.ch/t/payload-boosts-in-epbs/18769, it's essentially the only solution to a system of inequalities that guarantees builder and proposer safety. And it does so up to that value of \beta. For the PTC ownership there's a curve, for whatever stake the attacker has, what's the expected waittime until they own the PTC?.